test/aesgcmtest.c

```diff
@@ -121,16 +180,64 @@ static int ivgen_test(void)
     unsigned char iv_gen[16];
     unsigned char tag[32];
     unsigned char ct[32];
-    int ctlen = 0, taglen = 0;
+    unsigned char pt[32];
+    int ctlen = 0, ptlen = 0, taglen = 0;
+
+    return do_encrypt(iv_gen, NULL, 0, ct, &ctlen, tag, &taglen)
+           && do_decrypt(iv_gen, ct, ctlen, tag, taglen, pt, &ptlen);
+}
+
+static int chunk_256_test(void)
```

Comment:
I think this might be better done using evp_test.  Look at the data files in `test/recipes/30-test_evp_data`.
Generally, known answer tests are best located there.

Reply:
Ok, got it.

Reply:
Ok, I went through that and saw that there are plenty of such test cases like I added.
I will drop the commit with the additional test.

As it took me quite some time to get these tests properly running, here are the instructions:
```
OPENSSL_riscvcap="rv64gc_zbc_zbkb" QEMU_CPU="rv64,zbc=true,zbkb=true" qemu-riscv64 test/evp_test -iter 1 ./test/recipes/30-test_evp_data/evpciph_aes_common.txt
```